__tests__/seed.test.js

```diff
@@ -1,81 +1,70 @@
-const generator = require('../database/generateData.js');
+let generator = require('../database/generateData.js');
 const fs = require('fs');
+const options = {
+  autoClose: true,
+};
 
 describe('Data generation script', () => {
+
   test('can generate .csv file with arbitrary number of records', (done) => {
-    generator.write('seedData10.csv', 10)
-      .then(() => {
-        fs.readFile('../database/seedData10.csv', 'utf8', (err, data) => {
-          if (err) {
-            console.log('file read error: ', err);
-          } else {
-            const contents = data.trim().split('\n');
-            const regex = /[0-9][0-9]*,"[A-Za-z\s-,]*","{https:\/\/loremflickr.com\/3000\/2000\/sanfrancisco,https:\/\/loremflickr.com\/3000\/2000\/sanfrancisco,https:\/\/loremflickr.com\/3000\/2000\/sanfrancisco,https:\/\/loremflickr.com\/3000\/2000\/sanfrancisco,https:\/\/loremflickr.com\/3000\/2000\/sanfrancisco,https:\/\/loremflickr.com\/3000\/2000\/sanfrancisco,https:\/\/loremflickr.com\/3000\/2000\/sanfrancisco,https:\/\/loremflickr.com\/3000\/2000\/sanfrancisco,https:\/\/loremflickr.com\/3000\/2000\/sanfrancisco,https:\/\/loremflickr.com\/3000\/2000\/sanfrancisco}"/;
-            for (let i = 0; i < contents.length; i += 1) {
-              expect(contents[i].match(regex)).not.toBe(null);
-              if (i === 9) {
-                done();
-              }
+    generator.write('seedData50.csv', 50, fs.createWriteStream('seedData50.csv', options), () => {
+      fs.readFile('../database/seedData50.csv', 'utf8', (err, data) => {
+        if (err) {
+          console.log('file read error: ', err);
```

Comment:
I'm not 100% on this, but I believe if you catch the error, jest will not see that an error is thrown, and you may get a longer test due to done() never being called (instead it will time out). Perhaps it is better to throw if err exists.